packages/now-cli/src/index.js

```diff
@@ -1,5 +1,14 @@
 #!/usr/bin/env node
 
+try {
+  process.cwd();
+} catch (error) {
+  if (error && error.message && error.message.includes('uv_cwd')) {
+    console.error('The current working directory does not exist.');
```

Comment:
Let's use `output.error()` to get the error coloring?

Reply:
There is no output at this point because this is the first code that runs before any `require()` calls

Reply:
Looks like it works with `error` 👍